third_party/terraform/tests/resource_access_approval_folder_settings_test.go

```diff
@@ -9,9 +9,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
-func TestAccAccessApprovalFolderSettings_update(t *testing.T) {
```

Comment:
Can we add a comment to these stating where to look for the test runner? I remember being confused for the `AccessContextManagerServicePerimeter` tests, but the comment really helped!